dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyData.vue

```diff
@@ -10,8 +10,17 @@
       </div>
     </div>
     <div class="grid">
-      <div class="col-7">
-        <TaxonomyPanel :dataID="metaDataInfo.data[0].dataId" v-if="metaDataInfo.data.length > 0" />
+      <div
+        class="col-7"
+        v-if="metaDataInfo.data.length > 0 && metaDataInfo.data[0].dataType === 'EuTaxonomyDataForNonFinancials'"
```

Comment:
Was ist, wenn mehrere Frameworks reportet wurden und der erste Weder TaxoFinancial not TaxoNonFinancial ist?

Reply:
Schau ich mir jetzt an

Reply:
Ich habe jetzt implementiert, dass das erste DataSet, dessen Typ mit dem substring "EuTaxonomyDataFor" beginnt, genommen wird.

Reply:
Wird in DALA-450 behandelt